### PR TITLE
Desktop version PR

### PR DESCRIPTION
**Pull request by laurianerzb**
### Desktop version PR

#Added desktop nav inside html file
#Added media queries for desktop inside css
#linters errors Checked

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -355,3 +361,80 @@ textarea {
   border-radius: 8px;
   margin-left: 10px;
 }
+
+/* Desktop version QUERIES */
+@media (min-width: 764px) {
```
Comment (Abenezer-Tilahun) [inline styles.css:366]: - [ ] As requested in the requirements, please adjust your media query to 768px for the best responsiveness on all mobile devices. 😀
az

File: styles.css [hunk 2]
```diff
@@ -355,3 +361,80 @@ textarea {
...
+
+  .contact-section {
+    padding: 5% 25%;
+    margin-top: 5%;
+  }
+
+  .c-container h2 {
+    margin: 0;
+  }
+}
```
Comment (Abenezer-Tilahun) [inline styles.css:440]: - [ ] You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design For desktop size Look likes this : For contact section 
Capture

The Figma design For contact section Look likes this : 

Capture

- [ ] Kindly work on the contact section alignment for desktop size and adjust the position of "Get In Touch" button .

- [ Optional ] Kindly adjust the background color of "Headline" and "About Section" to white just like the figma design.

- [ ] The Minimum mobile screen size should be 375px. For making mobile screen size responsive for contact section, I recommend you to adjust it just like the figma design .

Your Contact section at the Minimum mobile screen size 375px : 
Capture

The Figma design at 375px
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @laurianerzb   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional   README
✔️ Good commit messages

almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @laurianerzb   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] As requested by the previous reviewer kindly make sure you `CONTACT SECTION`  is not affected, it still matches the provided Figma design. :wink: 

    - ### Your design

    image

    - ### Figma design
    image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @laurianerzb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost4

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

Kindly, I suggest you compare very well the original design of Figma with yours, I found these details to make your page look better
image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @laurianerzb , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**